rust/lance/src/io/writer/statistics.rs

```diff
@@ -150,12 +437,300 @@ fn get_boolean_statistics(arrays: &[&ArrayRef]) -> StatisticsRow {
     }
 
     StatisticsRow {
-        null_count,
+        null_count: ScalarValue::Int64(Some(null_count)),
         min_value: ScalarValue::Boolean(Some(true_present && !false_present)),
         max_value: ScalarValue::Boolean(Some(true_present || !false_present)),
     }
 }
 
+fn get_fixed_size_list_statistics<T: ArrowNumericType>(arrays: &[&ArrayRef]) -> StatisticsRow
+where
```

Comment:
let's remove this until later when we can think of what would be meaningful statistics for tensors.

Reply:
Removed.